Without this line, linker flags are placed before object files

diff --git a/dump2tar/src/Makefile b/dump2tar/src/Makefile
index a2ba26f..12f9034 100644
--- a/dump2tar/src/Makefile
+++ b/dump2tar/src/Makefile
@@ -22,6 +22,7 @@ check_dep_zlib:
 all: check_dep_zlib dump2tar
 
 dump2tar: $(core_objects) dump2tar.o $(libs)
+	$(CC) $(core_objects) dump2tar.o $(libs) -o $@ $(ALL_LDFLAGS) $(LIBEXECINFO_LIBS)
 
 install: dump2tar
 	$(INSTALL) -c dump2tar $(DESTDIR)$(USRBINDIR)
